modules/mw.cx.MachineTranslationService.js

```
@@ -189,75 +189,69 @@ mw.cx.MachineTranslationService.prototype.cxtoken_error = function () {
+		})
+		.catch(error => {
+			console.error('Error fetching mdwiki token:', error);
+		})
+	return result;
+};
+
+mw.cx.MachineTranslationService.prototype.fetchCxServerTokenMdwiki = function () {
+	const params = {
+		user: mw.user.getName(),
+		wiki: this.targetLanguage,
+		ty: "cxtoken"
```

ESLint

[error] Strings must use singlequote.

(quotes)